rfcs/044-browse-badges.md

```diff
@@ -0,0 +1,61 @@
+- Feature Name: Badges on landing page & browse view
+- Start Date: 2021-09-28
+- RFC PR: [amundsen-io/rfcs#44](https://github.com/amundsen-io/rfcs/pull/44)
+- Amundsen Issue: [amundsen-io/amundsen#0000](https://github.com/amundsen-io/amundsen/issues/0000) (leave this empty for now)
+
+# Badges on landing page & browse view
+
+## Summary
+
+This RFC proposes to introduce `Popular/All badges` section below `Popular tags` on landing page and & browse section.*
+
+! `Popular/All` is either this or that choice that has been explained in `Unresolved questions` section.
+
+## Motivation
+
+After introducing refactor of tags and badges only the former ones get dedicated section on landing page and . As badges
+are considered to be `curated` metadata, quick access to resources connected to particular badge is desired but currently
+missing. The only way to browse by badges is when users already `know` certain badge exists and uses advanced search
+ to search by badges. There is no place to actually find a list of badges used in Amundsen deployment nor to quickly filter
+ by one.
+
+From API perspective, metadata proxy already provides `badges` endpoint but it's not currently utilized.
+
+## Guide-level Explanation (aka Product Details)
+
+- Users can quickly access resources marked by badges by clicking on badge name on landing page
+- Users can browse badges in Amundsen using `Browse` view 
+
+## UI/UX-level Explanation
+
+- Landing page contains `Popular/All badges` section between `Popular tags` and `My Bookmarks`
+- Browse page contains `Badges` section below `Tags` section
+
+## Reference-level Explanation (aka Technical Details)
+
+- Introducing new component for showing list of badges on Landing page
+- Extending `Browse` view with badges section
+
+## Drawbacks
+
+I have not identified drawbacks to introducing this.
+
+## Alternatives
+
+- Leave current state as-is with users not being able to see all badges defined in Amundsen quickly.
+- Alternative UI approach would be to rename `Popular tags` section to `Popular labels` with `tags` and `badges` subsections
+
+## Prior art
+
+- This topic has been brought up by users several times on our Slack
+
+## Unresolved questions
+
+- Do we show `Popular badges` or just `Badges` (all badges) on landing page (number of unique badges might be significantly
```

Comment:
+1 to popular badges

Reply:
there is no 'count' attribute yet so for the first stage I'd go with `Available` badges